pkg/steps/release/create_release.go

```diff
@@ -103,16 +151,35 @@ func (s *assembleReleaseStep) Done() (bool, error) {
 }
 
 func (s *assembleReleaseStep) Requires() []api.StepLink {
-	return []api.StepLink{api.ImagesReadyLink()}
+	// if our prereq is provided, we don't need any prereqs
+	if len(os.Getenv(s.envVar())) > 0 {
```

Comment:
We should use `os.LookupEnv` for this?

Reply:
yeah, I'm still in the older habit.

Reply:
Actually, the value has to be greater than zero.  we ignore `RELEASE_IMAGE_INITIAL=` because having an empty env is invalid and if someone did bash to init the env var we want that.  We don't discriminate between empty and unset here.